src/routes/console/(billing-modal)/hoodieCover.svelte

```diff
@@ -14,13 +14,15 @@
 
     $: postText = encodeURIComponent(
         [
-            `Appwrite Pro is now available! `,
+            `@appwrite Pro just launched!`,
             ``,
-            `Why did I upgrade?`,
+            `I am one of the first to join🔥`,
             ``,
-            `Because`,
+            `Learn more about Appwrite Pro here: https://apwr.dev/AppwritePro`,
```

Comment:
I think this link should point here 
https://appwrite.io/blog/post/announcing-appwrite-pro